### ISSUE TITLE
[Bug Report] active-change in color-picker component should not emit when value when

### ISSUE THREAD

**Issue by zhangbobell** [ElemeFE/element#10901]
### Element UI version
2.3.6

### OS/Browsers version
macOS High Sierra 10.13.3/版本 66.0.3359.117（正式版本） （64 位）

### Vue version
2.5.16

### Reproduction Link
[http://jsfiddle.net/mmx38qxw/1858/](http://jsfiddle.net/mmx38qxw/1858/)

### Steps to reproduce
1. open [jsfiddle link](http://jsfiddle.net/mmx38qxw/1858/)
2. alert popup, indicate `active-change` event has emitted when mounted
3. click `change color` button, alert also popup, indicate when change `value` of `<el-color-picker>`, it also emit `active-change` event

The two scenes show the same bug: 

`active-change` should not emitted when we change the value in outer place

In other words, we assume that `active-change` event **should only** emitted when user pick color in the `sv-panel` or type in the `el-input` in dropdown panel

### What is Expected?
1. when mounted, color-picker will not emit `active-change` event
2. when click the  `change color` button, it also does not emit `active-change` event

### What is actually happening?
they both emit `active-change` event

**Comment by zhangbobell** [comment]
@Leopoldthecoder  I've came up with a pull request #10903, and fixed this problem, if there is any places to be updated, it will be my pleasure. thanks.

**Comment by Leopoldthecoder** [comment]
Oh thanks, I'll take a look tomorrow.
